src/providers/analytics.ts

```diff
@@ -73,12 +77,10 @@ export class AnalyticsEventBuilder {
   /**
    * Event handler that fires every time a Firebase Analytics event occurs.
    *
-   * @param {!function(!functions.Event<!functions.analytics.AnalyticsEvent>)}
-   *   handler Event handler that fires every time a Firebase Analytics event
+   * @param handler Event handler that fires every time a Firebase Analytics event
    *   occurs.
    *
-   * @return {!functions.CloudFunction<!functions.analytics.AnalyticsEvent>} A
-   *   Cloud Function you can export.
+   * @return A Cloud Function that you can export and deploy.
```

Comment:
Here too.  Later on I can do a cleanup for instances that we have missed.  Thanks!